src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,96 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+    private static final String EMPTY_JSON = "{}";
+
+    private final String fileDirectory;
+    private final String fileName;
+
+    /**
+     * Initialise the FileHelper with the directory and the file name
+     * @param fileDirectory the directory that the config file is located in
+     * @param fileName the name of the config file
+     */
+    public FileHelper(String fileDirectory, String fileName) {
+        this.fileName = fileName;
+        this.fileDirectory = fileDirectory;
+    }
+
+    /**
+     * Returns the File representation at the config file path
+     */
+    private Path getFilePath() {
+        return Paths.get(fileDirectory, fileName);
+    }
+
+    /**
+     * Attempts to create the config directory if it does not exist.
+     * @throws IOException When the config directory cannot be created
+     */
+    private void createDirectoryIfNonExistent() throws IOException {
+        File directory = new File(fileDirectory);
+        boolean directoryExists = directory.exists() && directory.isDirectory();
+        if (!directoryExists) {
+            if (directory.mkdirs()) {
+                logger.info("Config directory created: " + directory.toString());
+            } else {
+                logger.error("Could not create config file directory");
+                throw new IOException();
+            }
+        }
+    }
+
+    public boolean exists() {
+        return Files.exists(getFilePath());
+    }
+
+    /**
+     * Loads the content of a new file
+     * @return The String representation of the contents in a new file
+     */
+    public String loadNewFile() {
+        return EMPTY_JSON;
```

Comment:
And I think you would never load a new file? More like create new file. And loading new file should return empty string, which does not need a method inside `FileHelper` unless you are saving that empty file on the call.